packages/nc-gui-v2/composables/useViewData.ts

```diff
@@ -166,43 +165,41 @@ export function useViewData(
     }
   }
 
-  const updateRowProperty = async (row: Record<string, any>, property: string) => {
+  const updateRowProperty = async (row: Row, property: string) => {
```

Comment:
can we rename this argument to `toUpdate` or something? Using `toUpdate.row` would look cleaner than `row.row`

Reply:
sure :+1: